web-marketplace/src/pages/ChooseCreditClass/ChooseCreditClass.Grid.tsx

```diff
@@ -23,7 +31,17 @@ const ChooseCreditClassGrid: React.FC<React.PropsWithChildren<TemplateProps>> =
           borderTop: `1px ${theme.palette.grey[100]} solid`,
         })}
       >
-        <OnBoardingSection title="Choose a credit class">
+        <OnBoardingSection
+          title="Choose a credit class"
+          headerChildren={
```

Comment:
let's only display this in case the current user is a credit class issuer (which won't necessarily be the case when we'll deal with web2 users)